spec/components/title_spec.rb

```diff
@@ -55,4 +60,9 @@ def component_name
     render_component(title: 'Margin wat', margin_bottom: 17)
     assert_select "[class='^=govuk-\!-margin-bottom-']", false
   end
+
+  it "adds a lang attribute when set" do
```

Comment:
I'm not sure on the difference between the two tests you've added? Is one checking specifically because the context is now a link? If so, can we reflect that in the test name, to make it clear?

Reply:
Yeah, it was making sure it worked with and without a link, updated now.